src/develop/develop.c

```diff
@@ -1417,7 +1417,7 @@ static gboolean _dev_auto_apply_presets(dt_develop_t *dev)
            "        AND ?10 BETWEEN focal_length_min AND focal_length_max"
            "        AND (format = 0 OR format&?11!=0)"
            "        AND operation NOT IN"
-           "            ('ioporder', 'modulelist', 'metadata', 'export', 'tagging', 'collect', '%s'))"
```

Comment:
Not sure we want to remove modulelist from here since while testing the modulelist entries where still in my data.db.

Reply:
Read below, but indeed if we remove the modulelist as proposed below we can remove it from here.